db/migrate/20191122004444_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[6.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
Without having run this, I'd bet the issue you were getting here was a `column email already exists on table users`. That's because this syntax for [`change_table`](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_table) creates a new column